landing-page/content/common/quickstarts.md

```diff
@@ -0,0 +1,335 @@
+---
+url: quickstarts
+toc: true
+aliases:
+    - "quickstarts"
+    - "getting-started"
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+## <img src="../img/spark-menu-logo.png"> Spark and Iceberg Quickstart
+
+This quickstart will get you up and running with an Iceberg and Spark environment, including sample code to
+highlight some powerful features. You can learn more about Iceberg's Spark runtime by checking out the [Spark](../docs/latest/spark-ddl/) section.
+
+- [Docker-Compose](#docker-compose)
+- [Creating a table](#creating-a-table)
+- [Writing Data to a Table](#writing-data-to-a-table)
+- [Reading Data from a Table](#reading-data-from-a-table)
+- [Adding Iceberg to your Existing Spark Environment](#adding-iceberg-to-your-existing-spark-environment)
```

Comment:
I think these could be shorter, like "Adding Iceberg to Spark" and "Adding a catalog".

Reply:
Done! 👍